workflows/virology/influenza-isolate-subtyping/influenza-isolate-subtyping.ga

```diff
@@ -0,0 +1,2156 @@
+            "tool_version": "9.3+galaxy1",
+            "type": "tool",
+            "uuid": "cd5cfbd9-309a-42e5-957f-83258ef2a022",
+            "when": null,
+            "workflow_outputs": [
+                {
+                    "label": "per_segment_samples_combined_consensus",
+                    "output_name": "outfile",
+                    "uuid": "b7a61ef2-ddfa-402f-be33-a6afb461361f"
+                }
+            ]
+        },
+        "39": {
+            "annotation": "",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/rnateam/mafft/rbc_mafft/7.520+galaxy0",
+            "errors": null,
+            "id": 39,
+            "input_connections": {
+                "input|batches_0|inputs": {
+                    "id": 38,
+                    "output_name": "outfile"
+                },
+                "when": {
+                    "id": 11,
+                    "output_name": "output_param_boolean"
+                }
+            },
+            "inputs": [],
+            "label": null,
+            "name": "MAFFT",
+            "outputs": [
+                {
+                    "name": "outputAlignment",
+                    "type": "fasta"
+                }
+            ],
+            "position": {
+                "left": 4078.0509617367243,
+                "top": 185.3305934704922
+            },
+            "post_job_actions": {
+                "RenameDatasetActionoutputAlignment": {
+                    "action_arguments": {
+                        "newname": "Multiple sequence alignments per sample"
+                    },
+                    "action_type": "RenameDatasetAction",
+                    "output_name": "outputAlignment"
+                }
+            },
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/rnateam/mafft/rbc_mafft/7.520+galaxy0",
+            "tool_shed_repository": {
+                "changeset_revision": "bf28a8cff401",
+                "name": "mafft",
+                "owner": "rnateam",
+                "tool_shed": "toolshed.g2.bx.psu.edu"
+            },
+            "tool_state": "{\"datatype_selection\": {\"datatype\": \"--nuc\", \"__current_case__\": 1, \"scoring_matrix\": {\"type\": \"--kimura\", \"__current_case__\": 0, \"coefficient\": \"200\"}, \"fmodel\": false, \"gap_costs\": {\"use_defaults\": \"yes\", \"__current_case__\": 0}}, \"flavour\": {\"type\": \"mafft-fftns\", \"__current_case__\": 2}, \"input\": {\"mapping\": \"implicit\", \"__current_case__\": 0, \"batches\": [{\"__index__\": 0, \"inputs\": {\"__class__\": \"ConnectedValue\"}}]}, \"outputFormat\": \"\", \"reorder\": false, \"treeout\": false, \"__page__\": null, \"__rerun_remap_job_id__\": null}",
+            "tool_version": "7.520+galaxy0",
+            "type": "tool",
+            "uuid": "86f16717-083c-4cee-ac15-3620bf04c5c8",
+            "when": "$(inputs.when)",
+            "workflow_outputs": [
+                {
+                    "label": "per_sample_msas",
```

Comment:
```suggestion
                    "label": "per_segment_msas",
```

See above, but note that you will need to adjust the test if there's one that mentions this output.